### DISCUSSION TITLE
Üdvözöljük az eNyugta rendszer virtuális vita platformján

### DISCUSSION THREAD

**Discussion by renced42** [nav-gov-hu/eRECEIPT#1] [Általános]
## 👋 Üdvözöljük a virtuális eNyuga vita platformon!
 
  * Kövesse nyomon az eseményeket
  * Tegyen fel kérdéseket
  * Ossza meg ötleteit
  * Szavazon

Figyelem a hibabejelentéseket és egyéb technikai problémákat a [https://github.com/nav-gov-hu/eRECEIPT/issues](url) helyen jelezze!

**Comment by MicraMetripond** [comment]
Kedves NAV! Akkor jól értem, hogy a kérdéseket, egyenként, új témaként a kérdések és válaszok részhez kellene beírni? Vagy ott még lesznek létrehozva esetleg témakörök, hogy esetleg az azonos témában feltett kérdések, nagyjából egy helyen legyenek elkerülve azt, hogy ugyanaz a kérdés többször is előkerüljön? Köszönöm

**Reply by renced42** [reply]
Igen Célszerű egyenként a kérdéseket feltenni.
